docs/config/.updated.json

```diff
@@ -0,0 +1,5537 @@
+{
+  "index.html": {
```

Comment:
Did this used to be in another repo or was it dynamically generated all the time?

Reply:
I thought .updated.json was created and uploaded as a build artifact

Reply:
Are we talking about the index.html referenced in the .update.json file, or the json file itself?

Reply:
I interpreted Caleb's comment as a question on .updated.json file as a whole

Reply:
Must be the way I am building the docs locally. Removed.